### DISCUSSION TITLE
新版防沉迷

### DISCUSSION THREAD

**Discussion by Enaium** [Enaium/BiligameAddictionNotLimited#14] [Ideas]
新版防沉迷更新后，登录不限制失效了，现在还没解决方法。

**Comment by Enaium** [comment]
用没限制的账号登录，拦截所有请求，可正常登录；
用有限制的账号登录，却不可正常登录

**Reply by sudo-bai** [reply]
我有个方法，用adguard拦截请求，中间不退出的话可以保持到周一下午8点，到了这个时间就会报错（账号验证失败 12-4301）

**Reply by Enaium** [reply]
> 我有个方法，用adguard拦截请求，中间不退出的话可以保持到周一下午8点，到了这个时间就会报错（账号验证失败 12-4301）

这方法一只都能用

**Reply by Enaium** [reply]
> 我有个方法，用adguard拦截请求，中间不退出的话可以保持到周一下午8点，到了这个时间就会报错（账号验证失败 12-4301）

这个方法一直都能用

**Reply by sudo-bai** [reply]
> 

所以说有解决方案吗

**Reply by Enaium** [reply]
> > 
> 
> 所以说有解决方案吗

没有

**Reply by sudo-bai** [reply]
我有一个不成熟的想法，可不可以去解包直接修改登录器呢

**Comment by Enaium** [comment]
access_key这个东西应该是这样的

时间段内登录，认证服务器会生成一个access_key，之后返回到客户端
access_key这东西都在认证服务器里存着，会判断用户是否成年，或者是时间段内，不是就删除这个access_key
只有在线不被踢出的功能可行了

**Comment by Enaium** [comment]
服务器返回后，客户端会带着access_key调用native调用底层，估计那块就是unity那边的了
